msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -1,64 +1,111 @@
 ---
-title: Import External Platform Messages
-description: Learn how to use Microsoft Graph to import messages such as message history and data from an third-party platform to Teams.
+title: Import third-party platform messages to Teams using Microsoft Graph
+description: Learn how to use Microsoft Graph to import messages such as message history and data from any third-party platform to Teams.
 ms.localizationpriority: high
-author: "akjo"
+author: "surbhigupta"
 ms.topic: overview
-ms.owner: vishachadha
-ms.date: 09/02/2022
+ms.owner: mehakagarwal
+ms.date: 11/17/2025
 ---
 
 # Import third-party platform messages to Teams using Microsoft Graph
 
 With Microsoft Graph, you can migrate users' existing message history and data from an external system into a Teams channel. By enabling the recreation of a third-party platform messaging hierarchy inside Teams, users can continue their communications in a seamless manner and proceed without interruption.
 
 > [!NOTE]
-> In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data imported.
+> In the future, Microsoft might require you or your customers to pay extra fees based on the amount of data imported.
 
-## Import overview
+## Permissions
 
-At a high level, the import process consists of the following:
+| ScopeName             | DisplayName                        | Description                                      | Type             | Admin Consent? | Entities/APIs covered |
+|-----------------------|------------------------------------|-------------------------------------------------|------------------|----------------|-------------------------|
+| Teamwork.Migrate.All  | Manage migration to Microsoft Teams| Creating and managing resources for migration to Teams. | **Application-only** | Yes            | POST /team             |
```

Comment:
Link to the resource page: https://learn.microsoft.com/en-us/graph/permissions-reference#teamworkmigrateall